python/src/opendp/extras/numpy/make_np_clamp/__init__.py

```diff
@@ -49,7 +62,7 @@ def function(arg):
     return dp.t.make_user_transformation(
         input_domain,
         input_metric,
-        dp.np_array2_domain(**kwargs),
+        dp.numpy.np_array2_domain(**kwargs),
```

Comment:
If we change the path to introduce `numpy`, then `np_` in APIs is now redundant. I'd rather we drop `np_` from any APIs under `numpy` now, instead of changing APIs twice.